extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/videoQualityPanel.js

```diff
@@ -25,8 +28,13 @@ var VideoQualityPanel = React.createClass({
       selected: selectedBitrateId
     });
     this.props.togglePopoverAction();
+    this.props.toggleVideoQualityPanel();
   },
 
+    handleBackClick: function() {
+        this.props.toggleVideoQualityPanel();
+    },
+
   render: function() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has a complexity of 11 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)